variables.tf

```
@@ -66,3 +72,40 @@ variable "cloudwatch_event_rule_pattern_detail_type" {
   type        = string
   default     = "Security Hub Findings - Imported"
 }
+
+variable "enable_finding_aggregator" {
```

```suggestion
variable "finding_aggregator_enabled" {
```

this is our pattern to name all boll flags to enable/disable something using the `xxx_yyy_enabled` name